app/google-services.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
We should not add this file to github look at http://stackoverflow.com/questions/37358340/should-i-add-the-google-services-json-from-firebase-to-my-repository